pkg/server/auth/server.go

```diff
@@ -128,6 +131,11 @@ func NewOIDCConfigFromSecret(secret corev1.Secret) OIDCConfig {
 
 	cfg.Scopes = scopes
 
+	audiences := splitAndTrim(string(secret.Data["audiences"]))
+	if len(audiences) > 0 {
+		cfg.Audiences = audiences
```

Comment:
Should it be the clientId by default?

Reply:
(Alright, this piece of code is perhaps not the place to check for default though)

Reply:
actually, yes, that's a good idea!